FreeRTOS/FreeRTOS
# Add prvExerciseTaskHeapAllocationStats() to the MSVN Win32 and MPS2 Q…

Description
-----------
Add prvExerciseTaskHeapAllocationStats() to the MSVN Win32 and MPS2 QEMU IAR demos to exercise the function that track memory allocations on a task by task basis as introduced by https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/353.

Test Steps
-----------
Run "full" build of the MSVN Win32 demo and MPS2 QEMU IAR demos.

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/353


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
